goldie_test.go

```diff
@@ -56,10 +58,11 @@ func TestGoldenFileName(t *testing.T) {
 	}
 }
 
-func TestEnsureFixtureDir(t *testing.T) {
+func TestEnsureDir(t *testing.T) {
 	tests := []struct {
 		dir         string
 		shouldExist bool
+		fileExist   bool
```

Comment:
This file needs for be `gofmt`ed. The `bool` here is not aligned with the other types.

Reply:
it looks aligned?

Reply:
One space is missing to the left of `bool`

Reply:
Yeah - I'm not seeing that. It lines up right above, it lines up in my version, it lines up in my code editor, and I've got go fmt in both a save hook and pre-commit hook in my editor and it's not changing that file.

I mean, I could remove some spaces, but go fmt will just put them right back in?

Reply:
Alright, strange. I'll fix it when I merge it.